kubespawner/spawner.py

```diff
@@ -3010,7 +3044,14 @@ async def load_user_options(self):
             )
 
     async def _ensure_namespace(self):
-        ns = make_namespace(self.namespace)
+        if self.enable_user_namespaces:
+            ns = make_namespace(
+                self.namespace,
+                self._expand_all(self.user_namespace_labels),
+                self._expand_all(self.user_namespace_annotations),
+            )
+        else:
+            ns = make_namespace(self.namespace)
```

Comment:
I expect that this function isn't called unless `enable_user_namespace`, right?

I think that the `else` clause will never be executed and that this logic should reduce to

```suggestion
        ns = make_namespace(
            self.namespace,
            self._expand_all(self.user_namespace_labels),
            self._expand_all(self.user_namespace_annotations),
        )
```

Can you check if this guess is correct?

Reply:
You're right! I didn't notice that it's only called when enable_user_namespaces is enabled.

I removed the if statement